nextflow.config

```diff
@@ -10,10 +10,22 @@ vim: syntax=groovy
  * name here.
  */
 
+// Variable to specify the docker / singularity image tag to use
+// Picks up on use of -r 1.3 in nextflow command
+container_tag = workflow.revision ? workflow.revision : 'latest'
+wf_container = "scilifelab/ngi-rnafusion:${container_tag}"
+
+
+
+
 profiles {
   standard {
     includeConfig 'conf/uppmax.config'
   }
+  docker {
+    includeConfig 'conf/base.config'
```

Comment:
I think you're missing this file in the PR?

Reply:
You are correct, I will add it!